cli/api_client.go

```diff
@@ -0,0 +1,221 @@
+package cli
+
+import (
+	"github.com/chzyer/readline"
+	"io"
+	"strings"
+	"log"
+	"fmt"
+	"github.com/mysterium/node/identity"
+	"os"
+)
+
+func NewCliClient() *Client {
+	return &Client{
+		TequilaClient: NewTequilaClient(),
+	}
+}
+
+type Client struct {
+	TequilaClient *TequilaClient
+}
+
+func (c *Client) Run() {
+	completer := getAutocompleterMenu(c.TequilaClient)
+
+	rl, err := readline.NewEx(&readline.Config{
+		Prompt:          "\033[31m»\033[0m ",
+		HistoryFile:     "/tmp/mysterium-cli.log",
```

Comment:
1. Would be good to use `options.DirectoryRun`
2. Dont use `/` but `filepath.Join()` to cover different OSes